src/components/TrendChart.js

```diff
@@ -227,11 +233,24 @@ class TrendChart extends React.Component {
                       cx={x(datum.date)}
                       cy={y(datum.value.rate)}
                       fill={color(d.id)}
-                      r="3"
+                      r="3.5"
                     />
                   ))}
                 </g>
               ))}
+              {labels.map((d, i) => (
```

Comment:
@brendansudol thoughts on adding the sort statement to this line? My thinking is that it is more clear what the point of the sort is

```
  {labels
                .sort((a, b) => b.value.rate - a.value.rate)
                .map((d, i) => (
   ... 
```